### PR TITLE
Added support for HugeTlb controller

### PR DESCRIPTION
**Pull request by Zyqsempai**
Added support for new introduced HugeTlb controller https://www.spinics.net/lists/cgroups/msg24003.html

Signed-off-by: Boris Popovschi <zyqsempai@mail.ru>

### CODE REVIEW

**Comment by Zyqsempai** [comment]
@AkihiroSuda Fedora 31 doesn't have last kernel version with HugeTlb in it, fedora 32 is not out yet, do we have any other chance to run tests related to HugeTlb controller?

**Comment by AkihiroSuda** [comment]
Maybe we can use rawhide

**Comment by AkihiroSuda** [comment]
Or UML (hard to set up)

**Comment by AkihiroSuda** [comment]
cc @giuseppe

**Comment by giuseppe** [comment]
I am currently using a custom kernel for the hugetlb stuff, but I think Rawhide should already have it.

**Comment by Zyqsempai** [comment]
@AkihiroSuda Maybe do you have a vagrant box with RawHide? I can't find anything up to date, or maybe it make sense to migrate our CI to docker?

**Comment by AkihiroSuda** [comment]
This vagrant image may work: https://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Cloud/x86_64/images/
(Use https://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/COMPOSE_ID for substituting `Fedora-Rawhide-20200214.n.1` part)

**Comment by AkihiroSuda** [comment]
If updating the vagrant env to rawhide is difficult, you can skip the test in CI, but at least please do test on your local environment.

**Comment by Zyqsempai** [comment]
@AkihiroSuda So, i tried to make it work with rawhide in CI, but i failed. But i updated kernel on my machine, and it works fine.
So let's merge it like this.
If you will have some time here is the rawhide box i created https://app.vagrantup.com/zyqsempai/boxes/fedora-rawhide/versions/202002.24.0

File: v2/hugetlbv2_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCgroupv2HugetlbStats(t *testing.T) {
+	// This test will be temporary skipped, until the Fedora dist with kernel version 5.6 will be released.
+	t.Skip()
```
Comment (AkihiroSuda) [inline v2/hugetlbv2_test.go]: This should parse the controllers file and Skip() conditionally

Comment (Zyqsempai) [reply v2/hugetlbv2_test.go]: Done

**Comment by AkihiroSuda** [comment]
@fuweid PTAL

File: v2/stats/metrics.proto [hunk 2]
```diff
@@ -87,3 +88,9 @@ message IOEntry {
 	uint64 rios = 5;
 	uint64 wios = 6;
 }
+
+message HugeTlbStat {
+	uint64 current = 1;
+	uint64 max = 2;
+	string pagesize = 3;
+}
```
Comment (fuweid) [inline v2/stats/metrics.proto]: unexpected eof

**Comment by Zyqsempai** [comment]
@fuweid Fixed.

**Pull request merged**